tigerpath/majors_and_certificates/majors/SLA_2018.json

```diff
@@ -0,0 +1,123 @@
+      "course_list": [
+        "RUS 101",
+        "RUS 102",
+        "RUS 105",
+        "RUS 107"
+      ]
+    },
+    {
+      "name": "Required Courses",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "Concentrators are required to complete RUS 207 and one or more advanced language courses (RUS 208, 405, 406, 407, or 408; or one of the upper-level literature courses taught in Russian (SLA 308, 312, 350, 413). Students who place beyond RUS 207, or who complete the Russian Heritage Speakers courses (RUS 103-108), will work with the Departmental Representative to design an appropriate course of study.",
+      "req_list": [
+        {
+          "name": "Required",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "Concentrators are required to complete RUS 207.",
+          "course_list": [
+            "RUS 207"
+          ]
+        },
+        {
+          "name": "Advanced Language or Literature",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": "One additional upper-level language course: RUS 208, 405, 406, 407, 408 or one of the upper-level literature courses taught in Russian SLA 308, 312, 413",
+          "course_list": [
+            "RUS 208",
+            "RUS 405",
+            "RUS 406",
+            "RUS 407",
+            "RUS 408",
+            "SLA 308",
+            "SLA 312",
+            "SLA 413"
+          ]
+        },
+        {
+          "name": "Survey Courses",
+          "max_counted": 1,
+          "min_needed": 2,
+          "explanation": "Two of three survey courses: SLA 219, 220, 221",
+          "double_counting_allowed": true,
+          "course_list": [
+            "SLA 219",
+            "SLA 220",
+            "SLA 221"
+          ]
+        },
+      ]
+    },
+    {
+      "name": "Departmental Courses",
+      "max_counted": 1,
+      "min_needed": 8,
+      "explanation": "A minimum of eight departmental courses is required. Four upper-level (200 and above) courses must be within the department, two of which must be from the core survey courses (SLA 219, 220, 221); the other four courses may be from cognate areas depending on the student's particular interests. For example, if the major field of concentration is 19th-century prose, the program might include courses from French or German literature. Students with a strong interest in Russian and Soviet studies might take area courses in the Program in Russian and Eurasian Studies such as Russian history, politics, anthropology, or sociology.",
+      "req_list": [
+        {
+          "name": "Slavic Languages and Literatures",
+          "max_counted": null,
+          "min_needed": 4,
+          "explanation": "Four upper-level (200 and above) courses must be within the department, two of which must be from the core survey courses (SLA 219, 220, 221)",
+          "course_list": [

```

Comment:
Should explicitly encode the "two of which must be from the core survey courses (SLA 219, 220, 221)" part.